sdk/tests/integration/test_finbourne_identity.py

```diff
@@ -6,16 +6,16 @@ class IdentityTests(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        api_client_factory = ApiClientFactory(api_secrets_filename="secrets.json")
+        api_client_factory = ApiClientFactory()
         cls.users_api = api_client_factory.build(iapi.UsersApi)
         cls.roles_iapi = api_client_factory.build(iapi.RolesApi)
 
-    def test_roles(self):
+    def test_roles_are_returned(self):
```

Comment:
Could you please add a condition for when they are returned?

For example test_roles_are_returned_when_calling_list_roles
or test_roles_are_returned_when_roles_exist_on_account

choose one, and similar for users test